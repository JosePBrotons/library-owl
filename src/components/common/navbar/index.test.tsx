import I18n from 'i18n-js';
import React from 'react';
import { Text } from 'react-native';
import renderer from 'react-test-renderer';
import Navbar from '.';

test('renders card component correctly', () => {
    const navbarCmp = renderer
        .create(
            <Navbar>
                <Text>{I18n.t('global.settings')}</Text>
            </Navbar>
        )
        .toJSON();
    expect(navbarCmp).toMatchSnapshot();
});
