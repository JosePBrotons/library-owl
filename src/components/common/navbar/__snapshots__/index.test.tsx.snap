// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders card component correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgba(235, 246, 251, 1)",
      "height": 110,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/img/navbar.png",
      }
    }
    style={
      Object {
        "height": 111,
        "position": "absolute",
        "top": 0,
        "width": "100%",
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "marginVertical": 33,
        "paddingHorizontal": 20,
      }
    }
  >
    <View
      style={
        Object {
          "height": 20,
          "width": 20,
        }
      }
    />
    <Text>
      [missing "en.global.settings" translation]
    </Text>
    <View
      style={
        Object {
          "height": 20,
          "width": 20,
        }
      }
    />
  </View>
</View>
`;
